### PR TITLE
Update Foreman dependency: JWT

### PR DESCRIPTION
**Pull request by rahulbajaj0509**

### CODE REVIEW

**Comment by rahulbajaj0509** [comment]
This is a follow-up pr for https://github.com/theforeman/foreman-packaging/pull/4182, i hope it makes sense.

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -135,7 +135,7 @@ Requires: %{?scl_prefix}rubygem(get_process_mem)
 Requires: %{?scl_prefix}rubygem(rack-cors) >= 1.0.2
 Requires: %{?scl_prefix}rubygem(rack-cors) < 1.1.0
 Requires: %{?scl_prefix}rubygem(jwt) >= 2.1.0
-Requires: %{?scl_prefix}rubygem(jwt) < 2.2.0
+Requires: %{?scl_prefix}rubygem(jwt) = 2.2.1
```
Comment (ehelms) [inline packages/foreman/foreman/foreman.spec]: Needs a bump in the BuildRequires as well

Comment (rahulbajaj0509) [reply packages/foreman/foreman/foreman.spec]: Done. @ehelms it that correct?

Comment (ekohl) [reply packages/foreman/foreman/foreman.spec]: An exact match is usually incorrect. In general never pin this way. In general I use https://github.com/ekohl/theforeman-plumbing/blob/master/fm-packaging-update-deps to automatically do this.

Comment (rahulbajaj0509) [reply packages/foreman/foreman/foreman.spec]: @ekohl would running this script require https://github.com/theforeman/foreman-packaging/pull/4182 to merged first?

Comment (ekohl) [reply packages/foreman/foreman/foreman.spec]: No, the core PR which has just been merged.

**Pull request closed without merging**